website/docs/examples/tool_control/README.md

```diff
@@ -0,0 +1,330 @@
+---
+id: tool-control
+title: Tool Control
+---
+
+# Amiga Tool Control example
+
+:::danger Pre-release
+Running this example has minimum release versions of:
+
+- brain: `AmigaOS v2.3.0 +`
+- dashboard: `firmware v0.5.0 +`
+- PC: `farm-ng-amiga v2.3.0+`
+
+These may not yet be publicly available and are scheduled for release in early March 2024.
+To request early access, please reach out to us at: [support@farm-ng.com](mailto:support@farm-ng.com)
+:::
+
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. **Asynchronous Programming with asyncio**: Familiarity with Python's asyncio for writing concurrent
+code using the `async/await` syntax.
+3. **[farm-ng Canbus Service Overview](/docs/concepts/canbus_service/)**:
+This overview provides a base understanding of the gRPC service the client you create will connect to.
+4. **[Vehicle Twist example](/docs/examples/vehicle_twist/)**: It is recommended to have used the
+Vehicle Twist example for motion control of the Amiga before controlling the tools.
+:::
+
+:::caution Warning
+The tool control example will cause any tools connected to the Amiga to actuate when the dashboard
+is in auto mode.
+This includes H-bridges and PTOs.
+Make sure the area is clear before running examples.
+
+Unlike autonomous motion control, you cannot visualize the tool commands when the Amiga dashboard
```

Comment:
This will change, right?
Maybe just create a ticket so we don't forget to change it here in the future.

Reply:
This will not change

Reply:
I'm wrong, we have a ticket for this feature request

Reply:
I thought that this was the bug you described on the fw. But in that case, just disregard my previous comment.

Reply:
I will remove this comment because that feature should go into v0.5.0 :)